enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.tsx

```diff
@@ -39,6 +39,10 @@ export const _InteractiveQuestion = ({
   children = null,
 }: InteractiveQuestionProps &
   InteractiveQuestionResultProps): JSX.Element | null => {
+  if (!questionId) {
+    return <div>Loading...</div>;
+  }
```

Comment:
Should we use the (user-supplied) loader component we have instead?

Reply:
yes!